packages/lavapack/src/runtime.js

```diff
@@ -10007,6 +10008,15 @@ markVirtualizedNativeFunction)
           delete obj[prop];
         } catch (err) {
           if (prop in obj) {
+            if (typeof obj === 'function' && prop === 'prototype') {
+              obj.prototype = undefined;
+              if (obj.prototype === undefined) {
+                // eslint-disable-next-line @endo/no-polymorphic-call
+                console.log(`Tolerating undeletable ${subPath} === undefined`);
```

Comment:
whats being tested here?

Reply:
maybe just need to rebuild?

Reply:
or is this from updating ses?

Reply:
this is ses update

Reply:
this updated test allows me to successfully sandbox sentry behaviour in mm

https://github.com/MetaMask/metamask-extension/pull/15581/files#diff-595a6b34d8417a9d6ce54b84af7d484f46e5f57e3a5ff83729706b10ff1781a1R150-R164